src/tox/session/__init__.py

```diff
@@ -37,6 +37,9 @@
 from .commands.show_config import show_config
 from .commands.show_env import show_envs
 
+WAIT_INTERRUPT = 0.3
```

Comment:
why are these not using the config values?

Reply:
Here, `Action()` is created for internal purpose in the legacy builder, and the config values are only available in the envconfig.